modules/core/keeper/msg_server_test.go

```diff
@@ -483,41 +500,42 @@ func (suite *KeeperTestSuite) TestHandleTimeoutOnClosePacket() {
 		expPass  bool
 	}{
 		{"success: ORDERED", func() {
-			clientA, clientB, _, _, channelA, channelB := suite.coordinator.Setup(suite.chainA, suite.chainB, channeltypes.ORDERED)
-			packet = channeltypes.NewPacket(ibctesting.MockPacketData, 1, channelA.PortID, channelA.ID, channelB.PortID, channelB.ID, timeoutHeight, 0)
+			path.SetChannelOrdered()
+			suite.coordinator.SetupNew(path)
+			packet = channeltypes.NewPacket(ibctesting.MockPacketData, 1, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, timeoutHeight, 0)
 			counterpartyChannel = ibctesting.TestChannel{
```

Comment:
Why do you need to create this? Why can't I just call `path.EndpointB.CloseChannel()`

Reply:
yea, it'll be switched to that later